README.md

```diff
@@ -14,20 +14,27 @@ You cannot run the image directly because it does not come with the Pega web app
 
 The simplest way to use this image is to create your own Dockerfile with contents similar to the example below and extract the Pega distribution to the same directory as the Dockerfile.  It's a best practice to build this image on a Linux system to retain proper file permissions.  Replace the source paths with the actual paths to the Pega Infinity software libraries and specify a valid JDBC driver for your target database.
 
-    FROM pegasystems/pega-ready
+    FROM pegasystems/pega-ready as release
```

Comment:
Why is this a multi-stage build?  I only see one cascaded FROM here so I am not sure I follow.  It looks like we are calling pegasystems/pega-ready -> release -> unzipprweb?  The unzipprweb repo doesn't contain the war file does it - since it is really the pega-ready image?

Reply:
This is just an example set for the end-users on how to use "pega-ready" image. 
They will use pega-ready as the base image, but will have to add unzip package if they want to perform unzip of prweb which they will be mounting as per their logic in their DockerFile. 
Since, this secure image does not include all the packages, directly mounting prweb would fail. Hence, this example in README is just an example for them to add ONLY unzip package using multi stage process in DockerFile.